src/main/java/org/jabref/gui/preferences/ExternalTab.java

```diff
@@ -208,6 +248,11 @@ private void updateExecuteConsoleButtonAndFieldEnabledState() {
         consoleCommand.setDisable(!executeConsole.isSelected());
     }
 
+    private void updateExecuteFileBrowserButtonAndFieldEnabledState() {
+        fileBrowserButton.setDisable(!executeFileBrowser.isSelected());
```

Comment:
In JavaFX, you have the option to bind the properties once using `fileBrowserButton.disableProperty().bind(executeFileBrowser.selectedProperty().not()`. Then you don't need to take care of the state update yourself.